components/UserList.js

```diff
@@ -26,8 +26,10 @@ class UserList extends Component {
       return
     }
     const fetchedUsers = await secureFetch(this._getUrl(), accessToken)
-    if (fetchedUsers) {
-      this.setState({ users: fetchedUsers })
+    if (fetchedUsers.status === 'success') {
+      this.setState({ users: fetchedUsers.data })
+    } else {
+      window.alert(fetchedUsers.message)
```

Comment:
Generally, I don't think we want to be putting alerts everywhere in our code. I think this was just done as a shorthand early on, but this is not a good user experience. Will recommend change in PR.